plugins/lookup/consul_kv.py

```diff
@@ -2,6 +2,11 @@
 # (c) 2017 Ansible Project
 # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 from __future__ import (absolute_import, division, print_function)
+from ansible.module_utils._text import to_text
+from ansible.plugins.lookup import LookupBase
+from ansible.errors import AnsibleError, AnsibleAssertionError
+from ansible.module_utils.six.moves.urllib.parse import urlparse
+import os
```

Comment:
Can you please move the imports back? They do not belong here in Ansible modules.